test/carbonite/query_test.exs

```diff
@@ -185,6 +185,10 @@ defmodule Carbonite.QueryTest do
       assert [%Change{data: %{"name" => "Jack"}}] = changes(rabbit)
     end
 
+    test "can disable ordering", %{rabbit: rabbit} do
+      assert Map.get(Query.changes(rabbit, order_by: false), :order_bys) == []
+    end
```

Comment:
This test is great and IMO the best option to assert on "query is not ordered"...

Maybe add another test for custom ordering. The rabbit fixtures inserted in `insert_rabbits` have fixed `age` values, for instance.

Reply:
thanks, added this test